README.md

```diff
@@ -1,2 +1,87 @@
-# icepy
-Python-esque programming language with Icelandic grammar rules
+# IcePy
+
+A programming language that has strict Icelandic grammar rules built in.
+
+## Concepts
+
+There are a set of known variable names that can be used. Each have forms in
+nominative, accusative, dative, genitive.
+
+Each argument to a function must define the case expected.
+
+Each class has a gender
+
+Core constructs of the language may take different cases too.
+
+
```

Comment:
One concept left out but interested in opinions on: `language_mode`.

You have to define your own proficiency in the language and the interpreter acts accordingly.

Available modes are:
 - tourist
 - immigrant
 - citizen
 - native

At tourist mode, you can basically write normal python and everything works. In this mode, if you define anything in an IcePy way you are congratulated on your success.

Immigrant and above, the language gets more and more strict and things like the [`utlendigar`](https://github.com/alexcouper/icepy/pull/1/files#r116377848) mode no longer work.

For greater realism, occasionally the interpreter should fail on these modes because it itself doesn't speak IcePy but is just here for a summer job - but i think we can leave that out :p